netci.groovy

```diff
@@ -122,6 +122,8 @@ def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSet
             // params: Project, BaseTaskName, IsPullRequest (appends '_prtest')
             def jobName = Utilities.getFullJobName(project, config, isPR)
 
+            def testableConfig = buildType in ['debug']
+
             def infoScript = 'bash jenkins/get_system_info.sh'
             def debugFlag = buildType == 'debug' ? '--debug' : ''
```

Comment:
Specifically, @digitalinfinity indicated that there are plans to add a `test` config like we have on Windows, where we will change line 125 to be: `def testableConfig = buildType in ['debug', 'test']`